Art/Event/test_background.png.import

```diff
@@ -0,0 +1,34 @@
+[remap]
```

Comment:
There is no newline for the Event folder in the naming convention which makes tests fail
I would suggest adding the line:
Event : <name>
But this is a background, so maybe it should go in Art/Background
Or maybe we want to have a Background folder in the Event folder, in which case I need to modify the test for the naming convention